Description: prevent embedding of timestamps in files generated by rcc and uic
Origin: upstream, changesets 83ff904a1929 and a56bb17ad8ce
Last-Update: 2015-01-30

--- a/pyrcc/rcc.cpp
+++ b/pyrcc/rcc.cpp
@@ -45,7 +45,6 @@
 #include <qfile.h>
 #include <qfileinfo.h>
 #include <qlocale.h>
-#include <qdatetime.h>
 #include <qtextstream.h>
 #include <qbytearray.h>
 #include <qhash.h>
@@ -421,8 +420,7 @@
     fprintf(out, "# -*- coding: utf-8 -*-\n\n");
     fprintf(out, "# Resource object code\n");
     fprintf(out, "#\n");
-    fprintf(out, "# Created: %s\n", QDateTime::currentDateTime().toString().toUtf8().constData());
-    fprintf(out, "#      by: The Resource Compiler for PyQt (Qt v%s)\n", QT_VERSION_STR);
+    fprintf(out, "# Created by: The Resource Compiler for PyQt5 (Qt v%s)\n", QT_VERSION_STR);
     fprintf(out, "#\n");
     fprintf(out, "# WARNING! All changes made in this file will be lost!\n");
     fprintf(out, "\n");
--- a/pyuic/uic/__init__.py
+++ b/pyuic/uic/__init__.py
@@ -47,8 +47,7 @@
 
 # Form implementation generated from reading ui file '%s'
 #
-# Created: %s
-#      by: PyQt5 UI code generator %s
+# Created by: PyQt5 UI code generator %s
 #
 # WARNING! All changes made in this file will be lost!
 
@@ -149,7 +148,6 @@
     module is foo_rc.
     """
 
-    from time import ctime
     from PyQt5.QtCore import PYQT_VERSION_STR
 
     try:
@@ -159,7 +157,7 @@
 
     indenter.indentwidth = indent
 
-    pyfile.write(_header % (uifname, ctime(), PYQT_VERSION_STR))
+    pyfile.write(_header % (uifname, PYQT_VERSION_STR))
 
     winfo = compiler.UICompiler().compileUi(uifile, pyfile, from_imports, resource_suffix)
 
